specs/binding-http.spec/src/main/scripts/io/aklivity/zilla/specs/binding/http/streams/network/rfc7540/message.format/max.frame.size/client.rpt

```diff
@@ -94,21 +94,34 @@ read [0..20000]
 #
 # response DATA frame 9000 = max(30000 - 20000, 29000 - 20000)
 #
-read [0x00 0x23 0x28]                          # length = 9000
+read [0x00 0x1f 0x2e]                          # length = 7982
```

Comment:
I have adjusted the script based on the flow control fix, one thing is not clear to me why in the script we are setting 1000 as padding?